config/checkstyle-checks.xml

```diff
@@ -523,7 +523,15 @@
     <module name="StringLiteralEquality"/>
     <module name="SuperClone"/>
     <module name="SuperFinalize"/>
-    <module name="UnnecessaryParentheses"/>
+    <module name="UnnecessaryParentheses">
+      <property name="tokens"
+                value="EXPR, IDENT, NUM_DOUBLE, NUM_FLOAT, NUM_INT, NUM_LONG, STRING_LITERAL,
```

Comment:
I am not seeing where it was said the new tokens should not be added by default and why.

Reply:
@Kevin222004 ping

Reply:
as per https://github.com/checkstyle/checkstyle/pull/12703#discussion_r1164105414 it is decided to not support this three token by default so i have removed the token from getDefaultToken() and i have update this because it is showing me error if i dont add new token with all the existing one

Reply:
@rnveach please take look

Reply:
@rnveach ping